infrastructures/docker/infrastructure/src/test/java/org/eclipse/che/workspace/infrastructure/docker/server/mapping/ServersMapperTest.java

```diff
@@ -139,28 +132,28 @@ public void mapsServers(
             "exec-agent-ws",
                 new ServerImpl()
                     .withUrl("ws://" + hostname + ":32401/connect")
-                    .withAttributes(ATTRIBUTES_MAP),
-            "8000/tcp", new ServerImpl().withUrl("tcp://" + hostname + ":32000"),
-            "2288/udp", new ServerImpl().withUrl("udp://" + hostname + ":32288"))
+                    .withAttributes(ATTRIBUTES_MAP))
       },
       // mapping of internal servers
       {
-        mapOf(
-            "2288/udp", null,
-            "4401/tcp", null),
+        mapOf("4401/tcp", null),
         ImmutableMap.of(
             "ls-api", new ServerConfigImpl("4401", "tcp", null, INTERNAL_SERVER_ATTRIBUTE_MAP)),
         ImmutableMap.of(
             "ls-api",
             new ServerImpl()
                 .withUrl("tcp://" + machine + ":4401")
-                .withAttributes(INTERNAL_SERVER_ATTRIBUTE_MAP),
-            "2288/udp",
-            new ServerImpl().withUrl("udp://" + machine + ":2288"))
+                .withAttributes(INTERNAL_SERVER_ATTRIBUTE_MAP))
       }
     };
   }
 
+  private Map<String, String> mapOf(String key, String value) {
```

Comment:
ImmutableMap.of() ?

Reply:
It doesn't accept null values, which is needed to reproduce some cases